### DISCUSSION TITLE
panic: error parsing helm version 'v3.7.1+7.el8+g8f33223' goroutine 1 [running]:

### DISCUSSION THREAD

**Discussion by tharunamaddipudi** [helmfile/helmfile#692] [Questions] [answered]
Hi Team, getting below error for HELMFILE 0.150.0 and HELM v3.7.1+7 for below command
HELM VERSION
version.BuildInfo{Version:"v3.7.1+7.el8", GitCommit:"8f33223fe17957f11ba7a88b016bc860f034c4e6", GitTreeState:"clean", GoVersion:"go1.1
6.7"}
HELMFILE VERSION
Version 0.150.0
Git Commit fc634c9
Build Date 17 Jan 23 02:24 UTC (3 weeks ago)
Commit Date 17 Jan 23 00:34 UTC (3 weeks ago)
Dirty Build no
Go version 1.19.4
Compiler gc
Platform linux/amd64
$ helmfile sync
panic: error parsing helm version 'v3.7.1+7.el8+g8f33223'
goroutine 1 [running]:
[github.com/helmfile/helmfile/pkg/helmexec.New({0x242b19c]([http://github.com/helmfile/helmfile/pkg/helmexec.New(%7B0x242b19c)](https://github.com/helmfile/helmfile/pkg/helmexec.New(%7B0x242b19c)), 0x4}, 0x0, 0xc000013c08, {0x0, 0x0}, {0x2ac68c8?, 0xc0003d97a0})
/home/runner/work/helmfile/helmfile/pkg/helmexec/exec.go:120 +0x294
[github.com/helmfile/helmfile/pkg/app.(*App).getHelm(0xc0007540f0]([http://github.com/helmfile/helmfile/pkg/app.(*App).getHelm(0xc0007540f0)](https://github.com/helmfile/helmfile/pkg/app.(*App).getHelm(0xc0007540f0)), 0xc000a2c580)
/home/runner/work/helmfile/helmfile/pkg/app/app.go:782 +0x21e
[github.com/helmfile/helmfile/pkg/app.(*App).ForEachState.func1(0xc000a2c580?)](https://github.com/helmfile/helmfile/pkg/app.(*App).ForEachState.func1(0xc000a2c580?))
/home/runner/work/helmfile/helmfile/pkg/app/app.go:939 +0x3a
[github.com/helmfile/helmfile/pkg/app.(*App).visitStatesWithSelectorsAndRemoteSupport.func2(0xc000a2c580)](https://github.com/helmfile/helmfile/pkg/app.(*App).visitStatesWithSelectorsAndRemoteSupport.func2(0xc000a2c580))
/home/runner/work/helmfile/helmfile/pkg/app/app.go:1064 +0x6e
[github.com/helmfile/helmfile/pkg/app.(*App).visitStates.func1({0x7ffce23c26cf]([http://github.com/helmfile/helmfile/pkg/app.(*App).visitStates.func1(%7B0x7ffce23c26cf)](https://github.com/helmfile/helmfile/pkg/app.(*App).visitStates.func1(%7B0x7ffce23c26cf)), 0xd}, {0xc00098e0e0, 0x20})
/home/runner/work/helmfile/helmfile/pkg/app/app.go:889 +0x6f8
[github.com/helmfile/helmfile/pkg/app.(*App).visitStateFiles.func1()](https://github.com/helmfile/helmfile/pkg/app.(*App).visitStateFiles.func1())
/home/runner/work/helmfile/hel

**Comment by xiaomudk** [comment]
Hi, @tharunamaddipudi First of all, thank you for your feedback, I would like to know more about the following information:
1. What is your linux distribution?
2. How to install helm?

**Comment by yxxhero** [comment]
@tharunamaddipudi I guess that the helm binary is built by yourself. right?

**Comment by yxxhero** [comment]
normal output should be:
```
[root@devops eee]# ./helm version --short --client
v3.7.1+g1d11fcb
```

**Comment by yxxhero** [comment]
suggestions:
1. upgrade your helm version
2. use the official helm binary.

**Comment by yxxhero** [comment]
@xiaomudk I will use the new version parse function. when ready. please review.

**Comment by yxxhero** [comment, marked as answer]
@xiaomudk ready. https://github.com/helmfile/helmfile/pull/760 please go.
